.github/workflows/runChecks.yml

```diff
@@ -30,4 +30,7 @@ jobs:
         flake8 --config=flake8.ini
     - name: Test with nose
       run: |
-        py -m nose -sv --traverse-namespace -w src/tests
+        python -m nose -sv --traverse-namespace -w src/tests
+    - name: Validate NVDAVersion.json
+      run: |
+        python src/validate/validate.py NVDAVersions.json
```

Comment:
Rather than run these directly, I'd suggest using `tox`. It makes running the tests (in a segregated python virtual environment) as easy as running `tox`. This makes life easier for developers, they can easily run the tests in the same way as the system, less likely to make a mistake when typing out the command / getting the environment set up. It also prevents dependencies for tests from polluting the environment used to run the project (hopefully another virtual env). Since `tox` is so easy to use, and can configure multiple virtual environments (even testing multiple versions of python) some people even seem to use it to set up the environment for the project itself.